user_guide/common_features.rst

```diff
@@ -49,10 +49,32 @@ prior agreements.
 
 In OroCRM, all of this data is tracked using three types of records: channels, customers, and accounts.
 
+
+Understanding Accounts, Channels and Customers
+----------------------------------------------
+
+For each :ref:`channel created <user-guide-channel-guide-create>`, there is a set of 
+:term:`entities <Entity>` defined. These entities correspond to types of information collected from the corresponding 
+data source.
+
+One of the entities must represent a :term:`customer <Customer>`.
+
+Once a customer record is created, it is assigned to an account. Several accounts may 
+be :ref:`merged <user-guide-accounts-merge>` into one, regardless of the channels.(For example, if you have a B2B 
+customer that represents some client of yours, and then this client is also buying something from your Magento store).
```

Comment:
Edit: Once a customer record is created, it is assigned to an account. Several accounts may be :ref:`merged <user-guide-accounts-merge>` into one, regardless of the channels. (For example, when you have a B2B customer that represents some client of yours, and this client is also buying something from your Magento store.)